spring-pulsar-spring-cloud-stream-binder/src/test/java/org/springframework/pulsar/spring/cloud/stream/binder/PulsarMessageChannelBinderResolveSchemaTests.java

```diff
@@ -41,30 +46,33 @@
  */
 public class PulsarMessageChannelBinderResolveSchemaTests {
 
-	private SchemaResolver resolver = mock(SchemaResolver.class);
+	private final SchemaResolver resolver = mock(SchemaResolver.class);
```

Comment:
**[NIT]** Don't worry about removing now @cbornet - but we have avoided littering the code w/ finals. I know IDEA has warned that we could make this final. We landed on a guideline of making instance vars for immutable classes final and constants as well. Its mostly about consistency - IMO either we do it everywhere or not at all or in a set of prescribed cases that add value.

Reply:
done